wct-trusted-types-plugin/README.md

```diff
@@ -0,0 +1,29 @@
+# WCT Trusted Types Plugin
```

Comment:
Note that you can also set the CSP headers with <meta> tags, which might be easier.

See: https://lit.dev/playground/#project=W3sibmFtZSI6ImluZGV4Lmh0bWwiLCJjb250ZW50IjoiPCFET0NUWVBFIGh0bWw-XG48aGVhZD5cbiAgPG1ldGEgaHR0cC1lcXVpdj1cImNvbnRlbnQtc2VjdXJpdHktcG9saWN5XCIgY29udGVudD1cInJlcXVpcmUtdHJ1c3RlZC10eXBlcy1mb3IgJ3NjcmlwdCdcIiAvPlxuPC9oZWFkPlxuPGJvZHk-XG48c2NyaXB0PlxuICBjb25zdCBlbmZvcmNlZCA9ICgoKSA9PiB7XG4gICAgdHJ5IHtcbiAgICAgIGRvY3VtZW50LmNyZWF0ZUVsZW1lbnQoJ3AnKS5pbm5lckhUTUwgPSAnJztcbiAgICAgIHJldHVybiBmYWxzZTtcbiAgICB9IGNhdGNoIHtcbiAgICAgIHJldHVybiB0cnVlO1xuICAgIH1cbiAgfSkoKTtcbiAgZG9jdW1lbnQuYm9keS5hcHBlbmRDaGlsZChuZXcgVGV4dChgVHJ1c3RlZCB0eXBlcyAke2VuZm9yY2VkID8gJ2FyZScgOiAnYXJlIG5vdCd9IGVuZm9yY2VkIG9uIHRoaXMgcGFnZWApKTtcbjwvc2NyaXB0PlxuPC9ib2R5PlxuIn0seyJuYW1lIjoicGFja2FnZS5qc29uIiwiY29udGVudCI6IntcbiAgXCJkZXBlbmRlbmNpZXNcIjoge1xuICAgIFwibGl0XCI6IFwiXjIuMC4wXCIsXG4gICAgXCJAbGl0L3JlYWN0aXZlLWVsZW1lbnRcIjogXCJeMS4wLjBcIixcbiAgICBcImxpdC1lbGVtZW50XCI6IFwiXjMuMC4wXCIsXG4gICAgXCJsaXQtaHRtbFwiOiBcIl4yLjAuMFwiXG4gIH1cbn0iLCJoaWRkZW4iOnRydWV9XQ

Reply:
Oh, I totally forgot about that! I'll remove the plugin.

Reply:
Removed.